Makefile

```diff
@@ -3,12 +3,6 @@
 	cd lib_tflite_micro/submodules/tflite-micro && patch -p0 -i ../../../patches/tflite-micro.patch
 	touch .tflite_micro_patched.flag
 
-build: .tflite_micro_patched.flag
```

Comment:
How do we apply the patch now if it is being removed from the build here? Is it being done in a different manner?